packages/cli/src/commands/__tests__/test.test.js

```diff
@@ -19,7 +22,19 @@ jest.mock('@redwoodjs/structure', () => {
   }
 })
 
+jest.mock('fs')
+
+const MOCK_CONFIG_FILES = {
+  'api/jest.config.js': 'api jest config file',
+  'web/jest.config.js': 'web jest config file',
+}
```

Comment:
@Tobbe This is failing on Windows ('cause posix-only paths). Could you help guide @josemasar in the best direction toward the helpers we use to mock file system cross platform? Thanks in advance!

Reply:
@josemasar Since Tobbe won't be able to review until next week, if you have the time to look through CLI tests in general to see how cross-platform fixtures are handled I bet you'll find a path forward. For example, check out the `ensurePosixPath` export from the Internal package (I'm just not sure if this will work the same for fixtures).